lua/chadrc.lua

```diff
@@ -90,6 +90,12 @@ local M = {
             diffget_3 = "<leader>gl",
             git_blame = "<leader>gb"
         },
+        unix_keymap = {
```

Comment:
Function mentioned in my comment below should check the table in `line 9 (above)`, `options`